HedgeModManager/Game.cs

```diff
@@ -68,11 +68,29 @@ public static class Games
             CodesURL = Resources.URL_FML_CODES
         };
 
+        public static Game Tenpex = new Game()
+        {
+            GameName = "Puyo Puyo Tetris 2",
+            ExecuteableName = "PuyoPuyoTetris2.exe",
+            HasCustomLoader = true,
+            SupportsCPKREDIR = false,
+            ModLoaderDownloadURL = Resources.URL_TML_DL,
+            ModLoaderData = null,
```

Comment:
are you not gonna do an embedded loader for PPT2 with this PR?

Reply:
we're not going to have an embedded loader for PPT2, the exe becomes really big with it.

Reply:
ah